user-guide/Advanced_Functionality/Databases/STaaS/STaaS.md

```diff
@@ -21,13 +16,96 @@ Advantages of DataMiner Storage as a Service (STaaS) include:
 
 - Outstanding performance, scalability, and redundancy
 
-- Compatibility with all data types
-
 - Cost-effectiveness
 
 - Safe and secure data
 
 > [!TIP]
->
-> - To read more about Storage as a Service and its advantages, see [DataMiner STaaS: a game-changer for your storage needs](https://community.dataminer.services/dataminer-staas-a-game-changer-for-your-storage-needs/).
-> - As Storage as a Service is **scheduled for production in Q3 of 2023**, we are working on adding more information to this section. Check back for updates frequently!
+> If you have any questions or need support with the STaaS feature, contact <staas@dataminer.services>.
+
+## Setting up STaaS
+
+1. [Upgrade your DataMiner System](xref:Upgrading_a_DataMiner_Agent) to version 10.3.10 [CU1] or higher.
+
+   > [!IMPORTANT]
+   > We recommend always upgrading DataMiner to the latest available version to get the latest features and performance updates.
+
+1. Make sure your DataMiner System is [connected to dataminer.services](xref:Connecting_your_DataMiner_System_to_the_cloud).
+
+1. Retrieve your organization and coordination IDs:
+
+   1. [Open SLNetClientTest tool](xref:Opening_the_SLNetClientTest_tool).
+
+   1. [Connect to your DMS with SLNetClientTest tool](xref:Connecting_to_a_DMA_with_the_SLNetClientTest_tool).
+
+   1. Select *Advanced* > *CcaGateway* > *Get Global State*.
+
+   1. Note down your organization and coordination IDs.
+
+1. Make sure you have at least **DataMiner CloudGateway 2.8.0** installed on the system. See [Upgrading nodes to the latest DxM versions](xref:Managing_cloud-connected_nodes#upgrading-nodes-to-the-latest-dxm-versions).
+
+1. Contact your Skyline representative or <staas@dataminer.services> to register you system to use STaaS.
+
+1. Wait until the registration is completed.
+
+1. **Optionally**, contact your Skyline representative or <staas@dataminer.services> to migrate your existing data to STaaS.
+
+1. In the `C:\Skyline DataMiner` folder, open *DB.xml* and edit it to look like this:
```

Comment:
What about other DMAs in the cluster? Do you need to edit DB.xml on all of them? Or do you need to force sync the changes from this DMA to the others?

Reply:
needs to be changed in all DMAs